plugins/destination/postgresql/client/spec/spec.go

```diff
@@ -17,11 +17,11 @@ const (
 )
 
 type Spec struct {
-	// Connection string to connect to the database. This can be a URL or a DSN, for example:
+	// Connection string to connect to the database. This can be a URI or a DSN, for example:
```

Comment:
```suggestion
	// Connection string to connect to the database. This can be a DSN or a URI, for example:
```
Since we're prioritizing DSN, mention it first in the doc.

Reply:
Did it in another place forgot the spec, will fix